supreme/src/iosMain/kotlin/at/asitplus/signum/supreme/os/IosKeychainProvider.kt

```diff
@@ -328,8 +346,8 @@ sealed class IosSigner(final override val alias: String,
             { "Metadata type mismatch (RSA key, metadata not RSA) "}
 
             signatureAlgorithm = SignatureAlgorithm.RSA(
-                digest = resolveOption("digest", metadata.algSpecific.supportedDigests, config.rsa.v.digestSpecified, config.rsa.v.digest),
-                padding = resolveOption("padding", metadata.algSpecific.supportedPaddings, config.rsa.v.paddingSpecified, config.rsa.v.padding)
+                digest = resolveOption("digest", metadata.algSpecific.supportedDigests, config.rsa.v.digestSpecified, { config.rsa.v.digest }),
+                padding = resolveOption("padding", metadata.algSpecific.supportedPaddings, config.rsa.v.paddingSpecified, { config.rsa.v.padding })
```

Comment:
Use constants instead of inline strings

Reply:
why? it's part of the exception message, we don't usually pull debug messages from constants